### DISCUSSION TITLE
Image to Face

### DISCUSSION THREAD

**Discussion by Finnedsgang** [hacksider/Deep-Live-Cam#711] [General]
I don't know if this is already possible with the application, but is it possible to create a face model based on some pictures ? I created a digital human with AI and I would like to animate with my expression.

**Comment by hacksider** [comment]
if you mean creating a LORA, then yes. But you have to create multiple images on different lighting, background, expressions.

**Comment by Finnedsgang** [comment]
Thanks for the reply. Do you know How many of this Pictures  I will need ? Theres a walktrough option in the app ? 

Is it possible to use something like instantiD to generate them ?  

> Il giorno 10 ott 2024, alle ore 10:29, Kenneth Estanislao ***@***.***> ha scritto:
> 
> LORA

**Comment by hacksider** [comment]
There's no walkthrough on this since I just answer your question. Also, it all depends on the training model you need to create (which is outside this application use case already).
